bigbluebutton-html5/private/config/settings.yml

```diff
@@ -927,13 +927,9 @@ public:
       - 0.1
       - 0.2
     rtt:
-      - 500
-      - 1000
-      - 2000
-    level:
```

Comment:
Seeing this change raises the question -- where do you specify the jitter or audio loss levels? Do we still measure those?

Reply:
I do, I use it to show the connection status, I made the client consider the three of them to show the connection status, always showing the worts.